src/store/createStore.ts

```diff
@@ -25,7 +31,8 @@ export default function createStore(
       return state;
     },
     reset() {
-      state = {};
+      state = initialState;
+      dispatchListeners();
```

Comment:
`state = initialState` - because `reset` means that we clear our changes that we made in our `initialState`, we returns to our `initialState`, not to empty object, I think it is right

`dispatchListeners();` - because our state are changing, and we need to dispatch our listeners to update all connected components

Reply:
And I am using `reset` function to close my modal with preact application, by reseting the state to initial (remove all data and set `shown` property to `false`)

Reply:
You're 100% right. Thanks for that.